src/html/partials/concept.html

```diff
@@ -1,4 +1,73 @@
 <section class="concept-section">
-  <h2 class="sm-title">The Concept</h2>
-  <p class="bg-intro">Meet Florence</p>
+
+  <div class="concept-section__intro">
+    <h2 class="sm-title">The Concept</h2>
+    <p class="bg-intro">Meet Florence</p>
+
+    <p>Our explorations led to the concept of Florence — a personal companion that would help patients understand and manage their medication regimen. A chatbot at heart, patients would converse with her through text and picture messages.</p>
+      
+    <p>If developed, a simple, familiar interface would hide her powerful intelligence. Behind the scenes, machine learning and behavioral modeling would enable her to offer a tailored, human experience based on each patient’s needs.</p>
+  </div>
+
+  <ul class="concept-section__list">
+    <li class="concept-section__item">
+      <div class="concept-section__item__card">
+        <h3>She Gets To Know You.</h3>
```

Comment:
wondering why the `<h3>` doesn't have a class the same way that you have the `sm-title` on the `<h2>` above. We usually (maybe always?) use classes so that we are never tempted to use an `<h3>` or any other heading simply because it's the "right size".

Reply:
I think that semantically the `<h3>` is correct here and my thinking was to use descendent selectors on the `<h3>` and `<p>` tags in this section when I got to the css, but Jeremy mentioned the increased specificity and it sounds like this is just generally a bad practice. Going to use something like `sm-blue-title` instead (unless you have other thoughts).

Reply:
Yeah semantically here it is fine. I mean more broadly in the life of a project you may be tempted to use something out of place because that's just the style of a `h3`. As for your class name I would separate the "blue" from the "sm-title" but that's just my love of tailwind talking. Also keep in mind that if blue where to become green you'd have to change a whole bunch of classes. Maybe something more generic makes sense. Kinda up to you and how you like to do things.

Reply:
We also talked about giving the `--modifier` part of BEM a try, but I'm sure that'll get figured out once the css/styling part starts.